pkg/certificate/providers/tresor/fake/fake.go

```diff
@@ -107,7 +107,16 @@ func (c *fakeMRCClient) GetCertIssuerForMRC(mrc *v1alpha2.MeshRootCertificate) (
 		return nil, nil, err
 	}
 	issuer, err := tresor.New(ca, rootCertOrganization, 2048)
-	return issuer, pem.RootCertificate("rootCA"), err
+	if err != nil {
+		return nil, nil, err
+	}
+
+	cert, err := issuer.IssueCertificate("rootCA", 24*time.Hour)
```

Comment:
interesting, was this causing any bugs in tests?

Reply:
not currently, but I have some tests in an upcoming PR that will rely on a real CA cert being issued.